base/applications/regedit/treeview.c

```diff
@@ -441,42 +446,32 @@ static BOOL InitTreeViewImageLists(HWND hwndTV)
     HICON hico;       /* handle to icon  */
     INT cx = GetSystemMetrics(SM_CXSMICON);
     INT cy = GetSystemMetrics(SM_CYSMICON);
+    WCHAR szShell32[MAX_PATH];
 
     /* Create the image list.  */
     if ((himl = ImageList_Create(cx, cy, ILC_MASK | ILC_COLOR32, 0, NUM_ICONS)) == NULL)
         return FALSE;
 
     /* Add the open file, closed file, and document bitmaps.  */
-    hico = LoadImageW(hInst,
-                      MAKEINTRESOURCEW(IDI_OPEN_FILE),
-                      IMAGE_ICON,
-                      GetSystemMetrics(SM_CXSMICON),
-                      GetSystemMetrics(SM_CYSMICON),
-                      0);
+    wcscpy(szShell32, L"%SystemRoot%\\system32\\shell32.dll");
+    hico = NULL;
+    ExtractIconExW(szShell32, -IDI_SHELL_FOLDER_OPEN, NULL, &hico, 1);
```

Comment:
Actually this could be simplified by using the same `LoadImageW`, as shell32 should be already loaded in the process.